src/BestBooks.js

```diff
@@ -8,18 +12,55 @@ class BestBooks extends React.Component {
     }
   }
 
+
+
   /* TODO: Make a GET request to your API to fetch books for the logged in user  */
 
+  getBooks = async () => {
+    console.log(`${url}/books`);
+    let bookResults = await axios.get(url + '/books');
+
+    console.log(bookResults.data);
+    this.setState({
+      books: bookResults.data
+    });
+  }
+
```

Comment:
Good work on all the updates!

- For methods, getBooks, postBooks etc, consider adding in your try catch blocks so we can handle errors 👍